src/core/domain/models/taskList/taskList.entity.ts

```diff
@@ -0,0 +1,16 @@
+import TaskDto from "../task/dto"
+
+export default class TaskList {
+  
+  constructor(private list: TaskDto[]) {
+    this.list = []
+  }
+  add(object: TaskDto): void {
+    this.list?.push(object)
+  }
+
+  get(): TaskDto[] | undefined {
```

Comment:
Il faut en général éviter les méthodes get/set qui n'ont en réalité pas leur place en POO pure et dure.
https://www.infoworld.com/article/2073723/why-getter-and-setter-methods-are-evil.html